Sources/Purchasing/StoreKitAbstractions/SK2AppTransaction.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  Copyright RevenueCat Inc. All Rights Reserved.
+//
+//  Licensed under the MIT License (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      https://opensource.org/licenses/MIT
+//
+//  SK2AppTransaction.swift
+//
+//  Created by MarkVillacampa on 26/10/23.
+
+import StoreKit
+
+@available(iOS 15.0, tvOS 15.0, watchOS 8.0, macOS 12.0, *)
+internal struct SK2AppTransaction {
```

Comment:
Nice, good idea abstracting this. Let's add a docstring with something like "wrapper for `StoreKit.AppTransaction`".